src/main/java/com/solr/clientwrapper/infrastructure/entity/DocumentIndex.java

```diff
@@ -0,0 +1,34 @@
+package com.solr.clientwrapper.infrastructure.entity;
+
+
+
+import org.apache.solr.client.solrj.beans.Field;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.solr.core.mapping.SolrDocument;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@SolrDocument(collection = "user")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DocumentIndex {
```

Comment:
Why is this "user" collection hardcoded...

Reply:
that was old file .

Reply:
i have changes as you commented. and push on same branch